src/components/SeriesNav/SeriesNavTileContent.js

```diff
@@ -81,7 +89,11 @@ const SeriesNavTileContent = ({
   if (inline) {
     return (
       <LinkContainer>
-        <p {...styles.label} {...colorScheme.set('color', 'text')}>
+        <p
+          {...styles.label}
+          {...(current && styles.currentLabel)}
```

Comment:
these styles compete for priority regarding font style and `currentLabel` only wins if it's `css` call happens to come after the label css call. Which actually can't be guaranteed (same style leading to the same hash could be used else where and SSR css order is not stable)

Reply:
Two possible solutions, isolate font style into separate css calls:
```
  labelBase: css({
    marginTop: 0,
    overflow: 'hidden',
    textOverflow: 'ellipsis',
    whiteSpace: 'nowrap'
  }),
  labelRegular: css({
    ...sansSerifRegular12,
    [mUp]: {
      ...sansSerifRegular14
    }
  }),
  labelMedium: css({
    ...sansSerifMedium12,
    [mUp]: {
      ...sansSerifMedium14
    }
  }),
```

and then
```suggestion
          {...styles.labelbase}
          {...(current ? styles.labelMedium :  styles.labelRegular)}
```


Or use `merge` from glamor:
```suggestion
          {...merge(styles.label, current && styles.currentLabel)}
```
which guarantees the order – let's medium always win.